media_driver/media_top_cmake.cmake

```diff
@@ -123,7 +123,7 @@ endif(NOT DEFINED INCLUDED_LIBS OR "${INCLUDED_LIBS}" STREQUAL "")
 # post target attributes
 bs_set_post_target()
 
-if(MEDIA_RUN_TEST_SUITE)
+if(MEDIA_RUN_TEST_SUITE AND ${FULL_OPEN_BUILD} STREQUAL "no")
```

Comment:
Let's introduce a cmake native option rather than something self made: https://cmake.org/cmake/help/v3.0/command/option.html

This should be on a top level CMakeLists.txt I believe:
`option(FULL_OPEN_BUILD "Build fully open source driver (so not include binary kernels without sources)" OFF)`

After that all conditions will simply look like:
```
if (FULL_OPEN_BUILD)
endif()
```

And cmake command line:
`cmake -DFULL_OPEN_BUILD=ON ...`

Reply:
This comment was not addressed. Please, create a normal cmake option. Also, I suggest to change a n option name. Please, use ENABLE_KERNELS for it.

Reply:
First of all... is there a better word than "kernel"?
Every open source user will get really confused...